template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/s3guard/S3ConfigProviderTest.java

```diff
@@ -68,7 +80,8 @@ public void testGetHdfsServiceConfigsWithS3GuardWithAuthoriativeWarehousePath()
                 "<property><name>fs.s3a.s3guard.ddb.table</name><value>dynamoTable</value></property>" +
                 "<property><name>fs.s3a.s3guard.ddb.region</name><value>region</value></property>" +
                 "<property><name>fs.s3a.authoritative.path</name>" +
-                "<value>s3a://bucket/warehouse/managed</value></property>", sb.toString());
+                "<value>s3a://bucket/warehouse/managed</value></property>" +
+                "<property><name>fs.s3a.bucket.bucket.endpoint</name><value>s3.region.amazonaws.com</value></property>", sb.toString());
     }
```

Comment:
I suggest adding a test case that sets multiple FS locations with different buckets, so that the safety valve XML can contain more than one bucket endpoint overrides.

Reply:
I extended a test case to have to different buckets.